### PR TITLE
Inferno Program Edition v1.7.0

### PR DESCRIPTION
**Pull request by yunwwang**
- Adds validation for FHIR id data type specified in FHIR v4.0.1.
- Removes check within the Bulk Data Authorization test (BDA-03) that required systems to reject authorization requests with scopes other than system scopes specified in Bulk Data Access IG v1.0.0.
- Removes check within the Bulk Data Export tests (BDE-04 and BDE-05) that required systems to reject export requests with headers other than what specified in Bulk Data Access IG v1.0.0.
- Adds support for IE 11 in EHR Launch Test.
- Accepts HTTP response with status 401 as valid response for Bulk Data Authorization test (BDA-06).
- Fixes issue where headers with invalid characters in HTTP response could crash Inferno tests.
- Fixes the race condition in reference resolution tests.
- Fixes issue where MustSupport tests in Multi-Patient cause inadvertent effects of tests in Single-Patient setting.
- Adds the use parameter in Inferno's public keys that are used in the Multi-Patient API tests.

### CODE REVIEW

File: lib/app/views/index_onc_program.erb [hunk 1]
```diff
@@ -200,6 +200,32 @@ For alternate installation options, please visit <a href='https://github.com/onc
...
+    <div style="padding: 0 0 15px 0; font-size: 24px;">Inferno Program Edition v1.7.0</div>
+    <p>
+    The Inferno Program Edition v1.7.0 release fixes a number of issues that may incorrectly cause systems to fail tests.
+    It also includes a minor fixes not directly related to test execution.
+    This release may result in some systems that previously passed to now fail due to additional search tests.
+    </p>
+    <p>This version includes updates that:</p>
+    <ul>
+      <li>Adds test for validating FHIR id date type base on FHIR specification v4.0.1.</li>
+      <li>Removes check within the Bulk Data Authorization test (BDA-03) that required systems to reject authorization requests with scopes other than system scopes specified in Bulk Data Access IG v1.0.0</li>
```
Comment (arscan) [inline lib/app/views/index_onc_program.erb:213]: Be consistent with ending each bullet with a period.

File: lib/app/views/index_onc_program.erb [hunk 2]
```diff
@@ -200,6 +200,32 @@ For alternate installation options, please visit <a href='https://github.com/onc
...
+      <li>Removes check within the Bulk Data Export tests (BDE-04 and BDE-05) that required systems to reject export requests with headers other than what specified in Bulk Data Access IG v1.0.0</li>
+      <li>Adds support for IE 11 in EHR Launch Test.</li>
+      <li>Accepts HTTP response with status 401 as valid response for Bulk Data Authorization test (BDA-06).</li>
+      <li>Fixes issue where headers with invalid characters in HTTP response could crash Inferno tests.</li>
+      <li>Fixes the race condition in reference resolution tests.</li>
+      <li>Fixes issue where MustSupport tests in Multi-Patient cause inadvertent effects of tests in Single-Patient setting.</li>
+      <li>Adds the use parameter in Inferno's public keys.</li>
```
Comment (arscan) [inline lib/app/views/index_onc_program.erb:220]: may want to add something like '... that are used during the Multi-Patient API tests'

File: lib/app/views/index_onc_program.erb [hunk 3]
```diff
@@ -200,6 +200,32 @@ For alternate installation options, please visit <a href='https://github.com/onc
...
+
+    <div style="padding: 0 0 15px 0; font-size: 24px;">Inferno Program Edition v1.7.0</div>
+    <p>
+    The Inferno Program Edition v1.7.0 release fixes a number of issues that may incorrectly cause systems to fail tests.
+    It also includes a minor fixes not directly related to test execution.
+    This release may result in some systems that previously passed to now fail due to additional search tests.
+    </p>
+    <p>This version includes updates that:</p>
+    <ul>
+      <li>Adds test for validating FHIR id date type base on FHIR specification v4.0.1.</li>
```
[the remark this answers is not in the document]
Comment (yunwwang) [reply lib/app/views/index_onc_program.erb:212]: Changed to "Adds validation for FHIR id data type specified in FHIR v4.0.1"

**Review by arscan — APPROVED** [review]
These updates look good.  We can merge and do final review on inferno-qa before creating the release.

**Pull request merged**